e3db/base_crypto.py

```diff
@@ -52,17 +52,29 @@ def generate_keypair(self):
         # return public, private
         pass
 
+    @staticmethod
```

Comment:
I was just looking for pretty much exactly this function in the work I'm doing. Nice.